packages/block-editor/src/store/private-keys.js

```diff
@@ -3,3 +3,4 @@ export const globalStylesLinksDataKey = Symbol( 'globalStylesLinks' );
 export const selectBlockPatternsKey = Symbol( 'selectBlockPatternsKey' );
 export const reusableBlocksSelectKey = Symbol( 'reusableBlocksSelect' );
 export const sectionRootClientIdKey = Symbol( 'sectionRootClientIdKey' );
+export const editMediaEntityActionKey = Symbol( 'editMediaEntityActionKey' );
```

Comment:
It's a nitpick, but I think we could remove the word `entity` and `action` from the setting name. My reasoning is that those are concepts of the WordPress core data package, but the setting can be used my non-WordPress block editor builders injecting their own media APIs. If the setting were to stabilize we don't want those terms to be used.

That's the idea anyway, I don't know how well it works in practice. 😄

Reply:
Might be an idea to call it `mediaEdit`, so that it's similar to `mediaUpload` / `mediaSideload` settings.

Reply:
Too easy